app/models/feed.rb

```diff
@@ -104,4 +109,31 @@ def mime_type
     f = (audio_format || {})[:f] || 'mp3'
     AUDIO_MIME_TYPES[f]
   end
+
+  def copy_media(force = false)
+    feed_images.each { |i| i.copy_media(force) }
+    itunes_images.each { |i| i.copy_media(force) }
+  end
+
+  # API updates for feed_image=
+  def feed_image_file; feed_images.first; end
```

Comment:
Note: this API-getter returns the most recent image _of any status_.  So it's different than the `has_one :feed_image`.  Reasoning being - if you're working in the API, you expect the `{"feedImage":{"href":"/new/href"}}` you just PUT to show up immediately.  But we still want the RSS to contain the old image until processing completes.